tabpy/tabpy_server/state.ini

```diff
@@ -7,9 +7,15 @@ Access-Control-Allow-Headers =
 Access-Control-Allow-Methods = 
 
 [Query Objects Service Versions]
+PCA = {"description": "Returns the specified principal component", "type": "model", "version": 2, "dependencies": [], "target": null, "creation_time": 1565729597, "last_modified_time": 1565730202, "schema": null}

```

Comment:
remove this from state.ini pls

Reply:
This file shouldn't be in the repo at all, deleting.